lib/selenium.pm

```diff
@@ -0,0 +1,190 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017, 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Support for Selenium
+# Maintainer: Vladimir Nadvornik <nadvornik@suse.com>
+
+package selenium;
+use 5.018;
+
+use base 'Exporter';
+use Exporter;
+
+our @EXPORT = qw(
+  add_chromium_repos
+  install_chromium
+  enable_selenium_port
+  selenium_driver
+  wait_for_page_to_load
+  wait_for_link
+  wait_for_text
+  wait_for_xpath
+  select_input
+);
+
+use testapi;
+use utils 'zypper_call';
+
+use Selenium::Remote::Driver;
+use Selenium::Chrome;
+use Selenium::Waiter qw/wait_until/;
+use Selenium::Remote::WDKeys;
+
```

Comment:
Shouldn't `Selenium::Remote::WDKeys` be included in `cpanfile` as well?

Reply:
Added